src/sentry/replays/endpoints/project_replay_details.py

```diff
@@ -65,7 +102,27 @@ def get(self, request: Request, project: Project, replay_id: str) -> Response:
         else:
             return Response({"data": response[0]}, status=200)
 
+    @extend_schema(
+        operation_id="Retrieve a Replay Instance",
+        parameters=[
+            GlobalParams.ORG_SLUG,
+            GlobalParams.PROJECT_SLUG,
+            ReplayParams.REPLAY_ID,
+            ReplayValidator,
+        ],
+        responses={
+            202: RESPONSE_NO_CONTENT,
+            400: RESPONSE_BAD_REQUEST,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=ReplayExamples.GET_REPLAY_DETAILS,
```

Comment:
i'm wondering if this is example is off? if we delete we just return an empty response.

Reply:
it's copy pasta!

Reply:
can just say ` examples=None,` i believe